tests/unit_tests/test_wave_03.py

```diff
@@ -131,7 +131,11 @@ def test_swap_items_from_their_empty_returns_false():
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    try:
+        result == False
+    except ValueError as err:
+        print(f'An exception occurred. Here are the error details {err}')
```

Comment:
Instead of using try/except in a test, we'd expect to see `pytest.raises()` when testing to see that exceptions are deliberately raised.

Even if the appropriate exception is caught, it's not the best in terms of style, because the test result is less informative than it would be if you used `pytest.raises()`

Reply:
✅ Had some trouble but think we got it! Will push later!)